### PR TITLE
Bug fixes for issues #545, #570

### PR DESCRIPTION
**Pull request by Onariaginosa**
Bug fixes for issues #545, #570, and also changed workbook back to network in the export feature

Pull Request Checklist
- [x] Travis C.I. build passes
- [x] All four Demos look as expected when loaded
- [x] Reload works as expected
- [x] Loading file with Error brings up error modal, and it looks as expected
- [x] Loading file with Warning brings up warnings modal
- [x] Import works for both SIF and GraphML
- [x] Graph can be exported to SIF and GraphML
- [x] Print works as expected
- [x] Restrict graph to viewport works as expected (check/uncheck)
- [x] Viewport Size Changing works as expected (small/medium/large/fit)
- [x] Toggle between Grid Layout and Force Graph Layout works as expected
- [x] Force Graph Parameter Sliders change the number above them and have an effect on the graph
- [x] Locking/Unlocking/Resetting/Undo Resetting the force graph parameters works as expected
- [x] Enabling and disabling node coloring works as expected
- [x] Node coloring options work as expected (selection top/bottom dataset, averaging values, changing max value)
- [x] Weighted graph loaded with the "default to black edges" option checked appears as expected
- [x] Hide/Show Weights works as expected (always/never/upon mouseover)
- [x] Edge Weight Normalization Factor can be changed (set/reset)
- [x] Gray Edge Threshold can be changed, slider changes the number and has an effect on the graph
- [x] Checking Show Gray Edges as Dashed works as expected (check/uncheck)
- [x] D-pad left/right/top/bottom/center works as expected
- [x] Zoom slider changes number and has effect on graph (viewport and menu)
- [x] Right click on a node opens gene page, page is populated with correct data

### CODE REVIEW

**Comment by dondi** [comment]
At the 2/17 meeting we (re-)established a "developer QA" flow where if one developer posts a pull request, the other developer would, as part of their task for the week, go through the pull request check list. All PRs should now also include that checklist in the description https://github.com/dondi/GRNsight/wiki/Pull-Request-Checklist

**Comment by dondi** [comment]
To this effect @igreen1 we have set you as a "Reviewer" on the PR

**Comment by dondi** [comment]
At the 2/17 meeting we (re-)established a "developer QA" flow where if one developer posts a pull request, the other developer would, as part of their task for the week, go through the pull request check list. All PRs should now also include that checklist in the description https://github.com/dondi/GRNsight/wiki/Pull-Request-Checklist

File: server/controllers/network-sheet-parser.js [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+var constants = require(__dirname + "/workbook-constants");
+
+
+// const NETWORK_SHEET_NAMES = ["network", "network_optimized_weights"];
+
+// const isNetworkSheet = (sheetName) => {
+//     return NETWORK_SHEET_NAMES.some(function (network) {
+//         return (sheetName.toLowerCase() === network);
+//     });
+// };
```
Comment (dondi) [inline server/controllers/network-sheet-parser.js:18]: I get why these lines were just commented out and not outright deleted, but it looks like the next round can remove these entirely once this beta settles in.

File: server/controllers/semantic-checker.js [hunk 2]
```diff
@@ -172,7 +28,7 @@ var addWarning = function (workbook, message) {
     if (warningsCount < MAX_WARNINGS) {
         addMessageToArray(workbook.warnings, message);
     } else {
-        addMessageToArray(workbook.errors, errorList.warningsCountError);
+        addMessageToArray(workbook.errors, constants.errors.warningsCountError);
         return false;
     }
 };
```
Comment (dondi) [inline server/controllers/semantic-checker.js:34]: This appears to be a fairly large commented-out block. Let’s mark this for full deletion once this refactor has been put through more use.

**Review by dondi — APPROVED** [review]
Overall looks good from a code-review perspective—we’ll need to do the pull-request beta checklist before fully merging so @igreen1 if you can supply a review focusing on that, that would be great.

**Comment by igreen1** [comment]
All checks confirmed

Export has issues but this is a known bug

**Pull request merged**